src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -46,8 +48,6 @@ const styles = StyleSheet.create({
     backgroundColor: COLORS.anon_primary
   },
   textContainer: {
-    paddingLeft: BUBBLE_LEFT_PADDING + BUBBLE_LEFT_PADDING,
-    paddingRight: BUBBLE_RIGHT_PADDING,
     paddingTop: 4,
     paddingBottom: 4,
     borderRadius: 12,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The padding for the text container has been removed in the new hunk. If this was intentional and doesn't affect the layout, then it's fine. However, if the padding was necessary for proper alignment or spacing, you might want to add it back.
```diff
-    paddingLeft: BUBBLE_LEFT_PADDING + BUBBLE_LEFT_PADDING,
-    paddingRight: BUBBLE_RIGHT_PADDING,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->